src/Explorer/Menus/CommandBar/CommandBarComponentAdapter.tsx

```diff
@@ -29,7 +30,7 @@ export class CommandBarComponentAdapter implements ReactAdapter {
 
     // These are the parameters watched by the react binding that will trigger a renderComponent() if one of the ko mutates
     const toWatch = [
-      container.isPreferredApiTable,
+      userContext.apiType === "Tables",
```

Comment:
Unfortunately this doesn't work in this context. I'm not sure how the type system didn't flag it, but `toWatch` is an `Observable[]`, and this is just a plain `boolean`.

Reply:
ok i can remove